inc/breadcrumbs.php

```diff
@@ -80,11 +80,26 @@ function nightingale_uncanny_breadcrumb_check() {
 function nightingale_breadcrumb() {
```

Comment:
Would it be possible to change this function name to Hale without too much difficulty? Just so that we are slowly refactoring as we go with the theme.

Reply:
Done